.github/workflows/lint.yml

```diff
@@ -55,7 +57,7 @@ jobs:
       uses: DoozyX/clang-format-lint-action@v0.14
       with:
         extensions: 'c,h,cpp'
-        clangFormatVersion: 14
+        clangFormatVersion: 12
```

Comment:
Why downgrade? What is the effect?

Reply:
Downgrade mainly because the IIS system installation is version 12, and this allows us to point people to an existing installation that will match the CI's (meaning no formatting by hand required). There are minor formatting changes that come with this downgrade, but practically it imo doesn't matter.

Reply:
Ok, makes sense to have easy internal reproducibility.